pkg-manager/core/src/install/index.ts

```diff
@@ -591,28 +594,6 @@ Note that in CI environments, this setting is enabled by default.`,
     }
     /* eslint-enable no-await-in-loop */
 
-    function isWantedDepPrefSame (alias: string, prevPref: string | undefined, nextPref: string): boolean {
```

Comment:
This had to be moved up since it's called within `installCase`, but not defined by the time `installCase` gets called. We call `installCase` on line `580` above.